test/views/agent/HomePageViewSpec.scala

```diff
@@ -89,7 +89,9 @@ class HomePageViewSpec extends TestSupport with FeatureSwitching with ViewSpec {
       dunningLockExists,
       currentTaxYear,
       isAgent,
-      creditAndRefundEnabled = false
+      creditAndRefundEnabled = false,
+      paymentHistoryEnabled = isEnabled(PaymentHistoryRefunds)
```

Comment:
there should be tests with paymetnhistoryrefunds enabled and disabled, same as above